### PR TITLE
test: /bootstrap e2e test

### PR DESCRIPTION
**Pull request by lionelB**
add codecept with a first test using gherkin in french (https://codecept.io/bdd/#gherkin)
also add step_definitions base to have custom matchers
it use playwright under the hoods (https://codecept.io/playwright/#setup)

### CODE REVIEW

**Comment by lionelB** [comment]
En cas de fail des tests e2e, un screenshot est pris et est ajouté au artefact du pipeline (dans la vue `summary` https://github.com/SocialGouv/carnet-de-bord/actions/runs/1505173157)

File: .github/workflows/preproduction.yml [hunk 1]
```diff
@@ -84,3 +84,37 @@ jobs:
           token: ${{ secrets.GITHUB_TOKEN }}
           kubeconfig: ${{ secrets.KUBECONFIG }}
           rancherId: ${{ secrets.RANCHER_PROJECT_ID }}
+
+  ##############################################################################
+  ## End To End Tests running on all branche
```
Comment (Augustin82) [inline .github/workflows/preproduction.yml:89]:
```suggestion
  ## End To End Tests running on all branches
```

File: .github/workflows/review.yml [hunk 2]
```diff
@@ -96,3 +96,37 @@ jobs:
           kubeconfig: ${{ secrets.KUBECONFIG }}
           rancherId: ${{ secrets.RANCHER_PROJECT_ID }}
           socialgouvBaseDomain: ${{ secrets.SOCIALGOUV_BASE_DOMAIN }}
+
+  ##############################################################################
+  ## End To End Tests running on all branche
```
Comment (Augustin82) [inline .github/workflows/review.yml:101]:
```suggestion
  ## End To End Tests running on all branches
```

File: e2e/features/home.feature [hunk 3]
```diff
@@ -0,0 +1,13 @@
+#language: fr
+
+@home
+Fonctionnalité: Page d'acceuil
```
Comment (Augustin82) [inline e2e/features/home.feature:4]:
```suggestion
Fonctionnalité: Page d'accueil
```

File: e2e/features/home.feature [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+	Pour pouvoir me renseigner sur carnet de bord
+	En tant que visiteur
+	Je veux pouvoir consulter la page d'acceuil
```
Comment (Augustin82) [inline e2e/features/home.feature:7]:
```suggestion
	Je veux pouvoir consulter la page d'accueil
```

File: e2e/step_definitions/steps.js [hunk 5]
```diff
@@ -0,0 +1,145 @@
...
+
+Alors('je vois le thème {string}', (theme) => {
+	I.seeElement(`//a[text()="${theme}" and starts-with(@href, "/themes/")]`);
+});
+
+Alors('je ne vois pas le thème {string}', (theme) => {
+	I.dontSeeElement(`//a[text()="${theme}" and starts-with(@href, "/themes/")]`);
+});
+
+Alors('je suis redirigé vers la page: {string}', (url) => {
```
Comment (Augustin82) [inline e2e/step_definitions/steps.js:133]:
```suggestion
Alors('je suis redirigé vers la page : {string}', (url) => {
```

File: e2e/steps.d.ts [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+	interface SupportObject {
+		I: I;
+		current: any;
+	}
+	interface Methods extends Playwright {}
+	interface I extends ReturnType<steps_file> {}
+	namespace Translation {
+		interface Actions {
+			amOutsideAngularApp: 'suisALExtérieurDeLApplicationAngular';
+			amInsideAngularApp: 'suisALIntérieurDeLApplicationAngular';
```
Comment (Augustin82) [inline e2e/steps.d.ts:14]: question : Angular ?

**Review by Augustin82 — APPROVED** [review]

File: e2e/steps.d.ts (hunk 6 above)
Comment (lionelB) [reply e2e/steps.d.ts:14]: on peut supprimer ce fichier il sert pas finalement

**Review by gary-van-woerkens — APPROVED** [review]

**Pull request merged**